### ISSUE TITLE
Action does not fail if the firebase help text is shown

### ISSUE THREAD

**Issue by dyc3** [w9jds/firebase-action#26]
If this action is improperly configured, firebase prints out the help text and returns exit code 0. This causes github to think the action was successful, which is misleading.

**Comment by vural-ozturk** [comment]
Merhaba sizinle çalışmak eğlenceli olacak.

**Comment by w9jds** [comment]
Yeah, it is technically running a valid command and printing out a valid response. This is the same behavior much of the GitHub's official actions displayed when I originally created the action in alpha. I can take a look and see if I can change that behavior, but I'm not sure how important this is.
